src/lib.rs

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) 2019 King's College London created by the Software Development Team
+// <http://soft-dev.org/>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0>, or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, or the UPL-1.0 license <http://opensource.org/licenses/UPL>
+// at your option. This file may not be copied, modified, or distributed except according to those
+// terms.
+
+#![feature(alloc_layout_extra)]
+
+use std::alloc::{alloc, Layout};
+use std::mem::{size_of, transmute};
+use std::time::Instant;
+
+const REPS: usize = 100;
```

Comment:
So these are in-process iterations? Shall we name them as such?

Reply:
Agreed.